### ISSUE TITLE
Question on 8 Indicators of Verification Set and Image Visualization

### ISSUE THREAD

**Issue by Rmuli** [alexanderkroner/saliency#1]
Hi， Thank you for sharing the good idea and code.  The code can be run in **Python 3.5 and tensorflow 1.10.1** environments.  At the same time, I have a few questions to ask.

1: How to get the results of **8 indicators** in Table 1 and Table 2 on MIT300 test set and CAT2000 test set? I didn't see these eight indicators in the program.

2: How to **visualize the image** in the **validation set**? As shown in Figure 3 of the article. I don't seem to see the corresponding visualization programs or the results in **color maps**.

Can you give me some advice? 

Thanks !

**Comment by alexanderkroner** [comment]
Hey, thanks for letting me know! Regarding your questions:

1. Table 1 and 2 show the results I received after submitting my predictions to the MIT saliency benchmark (http://saliency.mit.edu). To compute the saliency metrics myself for the validation set, I used the MATLAB scripts here: https://github.com/cvzoya/saliency/tree/master/code_forMetrics

2. For the figures I simply took the saliency maps and overlaid the input images with them using a graphics editor. I did not use any code to do so.

I hope this answers your questions!
